src/dreyfus_index_manager.erl

```diff
@@ -86,6 +86,12 @@ handle_call({open_error, DbName, Sig, Error}, {OpenerPid, _}, State) ->
 
 handle_cast({cleanup, DbName}, State) ->
     clouseau_rpc:cleanup(DbName),
+    {noreply, State};
+
+handle_cast({move, DbName}, State) ->
```

Comment:
This doesn't match the gen_server:cast above and so it doesn't work. When I review code I am assuming that you have run through a successful move, but that doesn't seem possible with this problem.